linkml/generators/docgen/class_diagram.md.jinja2

```diff
@@ -2,57 +2,69 @@
 ```{{ gen.mermaid_directive() }}
  classDiagram
     class {{ gen.name(element) }}
+    click {{ gen.name(element) }} href "../{{gen.name(element)}}"
       {% for s in schemaview.class_parents(element.name)|sort(attribute='name') -%}
         {{ gen.name(schemaview.get_element(s)) }} <|-- {{ gen.name(element) }}
+        click {{ gen.name(schemaview.get_element(s)) }} href "../{{gen.name(schemaview.get_element(s))}}"
       {% endfor %}
 
       {% for s in schemaview.class_children(element.name)|sort(attribute='name') -%}
         {{ gen.name(element) }} <|-- {{ gen.name(schemaview.get_element(s)) }}
+        click {{ gen.name(schemaview.get_element(s)) }} href "../{{gen.name(schemaview.get_element(s))}}"
       {% endfor %}
       
       {% for s in schemaview.class_induced_slots(element.name)|sort(attribute='name') -%}
         {{ gen.name(element) }} : {{gen.name(s)}}
         {% if s.range not in gen.all_type_object_names() %}
           {{ gen.name(element) }} --> {{ s.range }} : {{ gen.name(s) }}
+          click {{ s.range }} href "../{{s.range}}"
```

Comment:
does this need a gen.name wrapper?

Reply:
Good catch @sierra-moxon! I fixed it in the most recent commit.